### PR TITLE
Added video links

### PR DESCRIPTION
**Pull request by tanevitt**
Added video links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ BattETL is a well-tested and an enterprise-ready python module for **E**xtractin
 
 <img src="./images/BattETL_Simple_System.svg">
 
+- [Installation & setup video](https://www.youtube.com/watch?v=tmudLhrj9xE)
```
Comment (chintanp) [inline README.md:7]: @tanevitt can we refactor the README a bit, so these links are under the complete navigation tree? 

A sub-section like "Videos" perhaps?

**Review by chintanp — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tanevitt) [reply README.md:7]: Sure. Do you want me to remove references to them elsewhere?

Comment (chintanp) [reply README.md:7]: I mean, move the references.

Comment (tanevitt) [reply README.md]: Done. Re-review requested.

File: README.md [hunk 2]
```diff
@@ -44,8 +45,18 @@ Not only does the BattETL extract and store all the battery test data, it also c
 
 As part of the transformation step BattETL calculates various cycle statistics (capacity, coulombic efficiency, CC/CV charge times, etc.) providing independent and supplemental cycle statistics to compliment those generated by the cycler software. These can particularly handy when easy-to-miss settings were overlooked in writing the schedule (e.g. forgetting to reset the capacity when incriminating the cycle.) Do you have some sort of hyper-boutique transformation or statistic calculation needed for a specific test? No problem. BattETL can apply user defined transformation functions too; all before the data is loaded to the database.
 
+# Video Guides
+
+The following are video guides BattETL and BattDB:
```
Comment (chintanp) [inline README.md:50]: video guides *for* BattETL and BattDB:

**Review by chintanp — APPROVED** [review]
LGTM! Thanks.

**Review by SyXuan — APPROVED** [review]
LGTM!

**Pull request merged**